src/GitHubCreateMergePRs/config.xml

```diff
@@ -6,19 +6,19 @@
     <merge from="dev16.0" to="release/dev16.1" />
     <merge from="release/dev16.1" to="release/dev16.2" />
     <merge from="release/dev16.2" to="release/dev16.3" />
-    <merge from="release/dev16.3" to="release/dev16.4-preview2" />
-    <merge from="release/dev16.4-preview2" to="release/dev16.4" />
-    <merge from="release/dev16.4" to="master" />
-
+    <merge from="release/dev16.3" to="release/dev16.4" />
+    <merge from="release/dev16.4" "release/dev16.5-preview1" />
+    <merge from="release/dev16.5-preview1" to="master" />
+      
     <!-- Roslyn branches (between vs-deps branches) -->
     <merge from="dev15.0.x-vs-deps" to="dev15.9.x-vs-deps" />
     <merge from="dev15.9.x-vs-deps" to="dev16.0-vs-deps" />
     <merge from="dev16.0-vs-deps" to="release/dev16.1-vs-deps" />
     <merge from="release/dev16.1-vs-deps" to="release/dev16.2" />
     <merge from="release/dev16.2" to="release/dev16.3-vs-deps" />
-    <merge from="release/dev16.3-vs-deps" to="release/dev16.4-preview2-vs-deps" />
-    <merge from="release/dev16.4-preview2-vs-deps" to="release/dev16.4-vs-deps" />
-    <merge from="release/dev16.4-vs-deps" to="master-vs-deps" />
+    <merge from="release/dev16.3-vs-deps" to="release/dev16.4-vs-deps" />
```

Comment:
these look wrong (most of these are vs-deps to vs-deps on the same release train, but others are not)

Reply:
This comment should be earlier in teh file `<!-- Roslyn does not have a dev16.2-vs-deps branch as it was merged back into dev16.2 -->`

Reply:
Fixed by removing 16.1 and 16.2 from code flow